src/pages/index.mdx

```diff
@@ -17,29 +17,33 @@ import { formatCompactNumber } from '@/utils/format'
 
 <BlogBanner /> 
 
-# Homebrew
+# 🖖 Homebrew
 
-Homebrew is an AI R&D studio. We work in the broad area of:
+Homebrew is a local AI company. 
 
-- Local AI
-- Small Language Models
-- Multi-modality
+We're a bunch of AI hobbyists tinkering with local AI. We're researching local AI, building tools, and contributing to open-source projects.
 
-We are the creators and lead maintainers of: 
+Our experiments so far:
 
-- 👋 [Jan](https://jan.ai): ChatGPT-alternative that runs 100% offline (~1.5m downloads)
+- 👋 [Jan](https://jan.ai): Local AI Assistant (~1.5m downloads)
 - 🤖 [Cortex](https://cortex.so): Local AI Engine
+- 🍓 Ichigo: Local Real-Time Voice AI
 
-We train models: 
+All our projects are built in public - we even train Ichigo in public.
 
-- 🦄 [llama3-s](https://github.com/janhq/llama3-s): Adding native listening ability to llama3
-
-We tinker with hardware:
+We have hardware dreams in the making:
 
 - 💡 Menlo: AI Hardware <span className="rounded-lg py-1 px-2 bg-gray-100 border border-gray-200 text-gray-700 dark:bg-gray-600 dark:text-gray-200 dark:border-gray-700 text-xs">Coming Soon</span>
 - ⛩️ Xanadu: GPU Cluster <span className="rounded-lg py-1 px-2 bg-gray-100 border border-gray-200 text-gray-700 dark:bg-gray-600 dark:text-gray-200 dark:border-gray-700 text-xs">Coming Soon</span>
 
-We are a fully remote team. See [open roles](https://homebrew.bamboohr.com/careers).
+We've got tons of lessons on:
```

Comment:
redundant. already mentioned above

Reply:
Yes, kinda but it's clearer for BD opportunities - it's for business reference to direct potential leads to our expertise, which is like helping people to make decisions we wish.